qtconsole/usage.py

```diff
@@ -0,0 +1,221 @@
+- ``C-l``: clear terminal.
+- ``C-a``: go to beginning of line.
+- ``C-e``: go to end of line.
+- ``C-u``: kill from cursor to the begining of the line.
+- ``C-k``: kill from cursor to the end of the line.
+- ``C-y``: yank (paste)
+- ``C-p``: previous line (like up arrow)
+- ``C-n``: next line (like down arrow)
+- ``C-f``: forward (like right arrow)
+- ``C-b``: back (like left arrow)
+- ``C-d``: delete next character, or exits if input is empty
+- ``M-<``: move to the beginning of the input region.
+- ``M->``: move to the end of the input region.
+- ``M-d``: delete next word.
+- ``M-Backspace``: delete previous word.
+- ``C-.``: force a kernel restart (a confirmation dialog appears).
+- ``C-+``: increase font size.
+- ``C--``: decrease font size.
+- ``C-M-Space``: toggle full screen. (Command-Control-Space on Mac OS X)
+
+The pager
+=========
+
+The Jupyter QtConsole will show long blocks of text from many sources using a
+builtin pager. You can control where this pager appears with the ``--paging``
+command-line flag:
+
+- ``inside`` [default]: the pager is overlaid on top of the main terminal. You
+  must quit the pager to get back to the terminal (similar to how a pager such
+  as ``less`` or ``more`` works).
+
+- ``vsplit``: the console is made double-tall, and the pager appears on the
+  bottom area when needed.  You can view its contents while using the terminal.
+
+- ``hsplit``: the console is made double-wide, and the pager appears on the
+  right area when needed.  You can view its contents while using the terminal.
+
+- ``none``: the console never pages output.
+
+If you use the vertical or horizontal paging modes, you can navigate between
+terminal and pager as follows:
+
+- Tab key: goes from pager to terminal (but not the other way around).
+- Control-o: goes from one to another always.
+- Mouse: click on either.
+
+In all cases, the ``q`` or ``Escape`` keys quit the pager (when used with the
+focus on the pager area).
+
+Running subprocesses
+====================
+
+The Jupyter QtConsole uses the ``pexpect`` module to run subprocesses
+when you type ``!command``.  This has a number of advantages (true asynchronous
+output from subprocesses as well as very robust termination of rogue
+subprocesses with ``Control-C``), as well as some limitations.  The main
+limitation is that you can *not* interact back with the subprocess, so anything
+that invokes a pager or expects you to type input into it will block and hang
+(you can kill it with ``Control-C``).
+
+We have provided as magics ``%less`` to page files (aliased to ``%more``),
+``%clear`` to clear the terminal, and ``%man`` on Linux/OSX.  These cover the
+most common commands you'd want to call in your subshell and that would cause
+problems if invoked via ``!cmd``, but you need to be aware of this limitation.
+
+Display
+=======
+
+The Jupyter QtConsole can now display objects in a variety of formats, including
+PNG and SVG. This is accomplished using the display functions in
```

Comment:
Again, this should make it clear that it's an example of how to do it from IPython, and that it may be different for other kernels.

Also, I think we can lose the word 'now' in the line above.